src/subgraph/queries.ts

```diff
@@ -18,6 +18,23 @@ export const getTokenSalesForNftQuery = gql`
   }
 `;
 
+export const getBuyNowTokenSales = gql`
+  query DomainTokenSolds($tokenId: String!, $contractAddress: String!) {
```

Comment:
in the other queries in this file $contract is of type `Bytes!` but this is of type `String!`. Could this be an issue?

Reply:
I saw this but left it as `String!` to match the typing in the subgraph schema file, and it is working fine